test/hotspot/jtreg/runtime/memory/ReadFromNoaccessArea.java

```diff
@@ -62,6 +62,7 @@ public static void main(String args[]) throws Exception {
     if (output.getStdout() != null && output.getStdout().contains("WB_ReadFromNoaccessArea method is useless")) {
       throw new SkippedException("There is no protected page in ReservedHeapSpace in these circumstance");
     }
+    output.shouldNotHaveExitValue(0);
```

Comment:
This is redundant. You will never have a zero exit value if you match the crash messages or else throw the exception. Seeing the crash message implies the return code is not zero so checking is a waste of time IMO not a more rigorous check. If the test did not crash then we won't match the expected output and we will still fail.

Reply:
Sorry, please ignore my previous comment. If we inadvertently caused the call to exit with a zero return in case of a crash, this change would detect it.